todos/week2/10_functions_and_scopetodo.py

```diff
@@ -1,7 +1,7 @@
 """
 Working with functions and scope
 """
-
```

Comment:
There should be a space between the """ on line 3 and the start of the section. I can't tell if you got rid of it, but it should go Line 3: """ Line 4:          Line 5: TODO: Section 1: